substrate/launcher/main.go

```diff
@@ -145,6 +145,14 @@ func main() {
 	// Unset any RUBYLIB, we don't want this bleeding into our runtime
 	newEnv["RUBYLIB"] = ""
 
+	if runtime.GOOS == "darwin" {
```

Comment:
@mitchellh might have ideas about how to make this faster, but since it's OSX maybe we don't care.